docs/GCS-connector.md

```diff
@@ -0,0 +1,35 @@
+# Google Cloud Storage Connector
+
+Description
+-----------
+This plugin can be used to browse and sample from Google Cloud Storage.
+
+Cloud Storage allows world-wide storage and retrieval of any amount of data at any time.
+You can use Cloud Storage for a range of scenarios including serving website content,
+storing data for archival and disaster recovery,
+or distributing large data objects to users via direct download.
+
+Credentials
+-----------
+If the plugin is run on a Google Cloud Dataproc cluster, the service account key does not need to be
```

Comment:
This section should be modified, as connectors don't run on Dataproc.

Reply:
I think we can shorten and move this section to the Service Account part of the properties. Something like:

When running on Google Cloud Platform, the service account key does not need to be provided, as it can automatically be read from the environment. In other environments, the service account key must be provided.